lib/exchangeInstance.js

```diff
@@ -7,11 +7,11 @@ import {createRequire} from 'node:module';
 import {encodeId, decodeId, generateId} from './helpers.js';
 import {getAppIdentity} from '@bedrock/app-identity';
 import {httpsAgent} from '@bedrock/https-agent';
+import {Ed25519Signature2020} from '@digitalbazaar/ed25519-signature-2020';
+import {ZcapClient} from '@digitalbazaar/ezcap';
```

Comment:
Sort imports alphabetically.

Reply:
I used the newer listing rules to alphabetize them: https://github.com/digitalbazaar/bedrock-vc-exchanger/pull/10/commits/af871c95b640584170d0a89b8179c1c3d557c090